test/base/EscrowInitializeTest.js

```diff
@@ -0,0 +1,90 @@
+// JS Libraries
+const { withData } = require('leche')
+const { t } = require('../utils/consts')
+const { createTestSettingsInstance } = require('../utils/settings-helper')
+const LoansBaseInterfaceEncoder = require('../utils/encoders/LoansBaseInterfaceEncoder')
+const { encodeLoanParameter } = require('../utils/loans')
+
+// Mock contracts
+const Mock = artifacts.require('./mock/util/Mock.sol')
+const DappMock = artifacts.require('./mock/base/escrow/dapps/DappMock.sol')
+const DAI = artifacts.require('./mock/tokens/DAIMock.sol')
+
+// Smart contracts
+const Escrow = artifacts.require('./mock/base/EscrowMock.sol')
```

Comment:
As we are testing Escrow contract, it would be better to use native contract (not mock) Escrow and mock only other contracts, unless there is a specific need to test using a mock instance.